src/test/java/net/sf/jabref/logic/layout/LayoutTest.java

```diff
@@ -86,15 +82,6 @@ public void testHTMLChar() throws IOException {
     }
 
     @Test
-    @Ignore
```

Comment:
I think, we decided to remove newlines in non-multiline fields. Is it https://github.com/JabRef/jabref/pull/357?

Even if that is not the desired behavior, I would add `// TODO: The correct behavior is` to indicate that someone should work on it.

Alternative: open an issue on GitHub. - I am currently optimizing (i) to keep the number of opened issues low and (ii) to have documentation close to the code and not spread around.